database/insertData/03_template_A_featureShowcase.js

```diff
@@ -456,6 +456,8 @@ exports.queries = [
                         label: "This Radio button group has no default"
                         options: ["Option A", "Option B", "Option C"]
                         # Testing no default
+                        hasOther: true
```

Comment:
Oh! So we have this `hasOther` I think I made a comment to remove this, so please ignore my comment - I think you might want to move that documentation change to this PR?